zebedee-cms/src/main/java/com/github/onsdigital/zebedee/ZebedeeConfiguration.java

```diff
@@ -275,6 +271,21 @@ private Keyring initCentralKeyring() throws KeyringException {
         return centralKeyring;
     }
 
+    private StartUpAlerter initStartUpAlerter() {
+        List<String> startUpNotificationRecipients = slackChannelsToNotfiyOnStartUp();
+
+        if (startUpNotificationRecipients == null || startUpNotificationRecipients.isEmpty()) {
+            return new NoOpStartUpAlerter();
+        }
+
+        SlackClient slackClient = new SlackClientImpl(new Profile.Builder()
+                .emoji(":flo:")
+                .username("Florence")
+                .authToken(System.getenv("slack_api_token"))
```

Comment:
Do we need to protect against this being null too? I guess this was the main problem as I ran the code locally successfully with a null channels list, but I already had a slack token, suggesting that it was the missing slack token rather than channel list that was Angela's issue possibly.

Reply:
Ah, or maybe because at that point I hadn't actually signed i to zebedee. As you were.